recipes/sphinxcontrib-getthecode/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "sphinxcontrib-getthecode" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FabriceSalvaire/sphinx-getthecode/archive/v{{ version }}.tar.gz
+  # run openssl sha256 <file name>
+  sha256: 9e84b57b36f07d6dbe157521cc9033cb491142af7dc45b277bf608f8e5b84eda
+  # git_url: https://github.com/FabriceSalvaire/sphinx-getthecode
+  # git_rev: ... # "v{{ version }}"
+  # git_depth: 1 # (Defaults to -1/not shallow)
+
+build:
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<34]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  has_prefix_files:
+  always_include_files:
+
+requirements:
+  build:
+  host:
+    - python
```

For using `noarch`, constrain Python like so

```suggestion
    - python >=3.4
```